### PR TITLE
Blog post to announce 2023 community survey.

### PR DESCRIPTION
**Pull request by iskerrett**
A blog post to announce a new Node-RED community survey.

### CODE REVIEW

File: _posts/2023-02-24-community-survey [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+The survey will be open until March 31. Of course we would like to get as many respondents
+as possible. In 2019, we have over 800 individuals respond so hopefully we can meet this number
+again this year. 
+
+Once the survey has been completed, we will publish the results for the entire community to see.
+
+The survey should take less than 10 minutes to complete. Thank you in advance for your feedback.
+
+[Survey Form](https://forms.gle/yhn8EzxA98yDFsjh7)
```
Comment (kazuhitoyokoi) [inline _posts/2023-02-24-community-survey:23]: Screenshot 2023-02-23 at 18 41 02
"Raspbian" -> "Raspberry Pi OS"

Comment (iskerrett) [reply _posts/2023-02-24-community-survey:23]: Change made. Thanks for the feedback.

Comment (kazuhitoyokoi) [reply _posts/2023-02-24-community-survey:23]: Thank you for updating it. But it seems incorrect...
"Raspbian Pi OS" -> "Raspberry Pi OS"
Screenshot 2023-02-24 at 1 22 53

Comment (knolleary) [reply _posts/2023-02-24-community-survey:23]: Fixed. Thanks @kazuhitoyokoi

Comment (kazuhitoyokoi) [reply _posts/2023-02-24-community-survey:23]: Another one, "Windews Desktop" -> "Windows Desktop".

Comment (knolleary) [reply _posts/2023-02-24-community-survey:23]: Done

Comment (kazuhitoyokoi) [reply _posts/2023-02-24-community-survey:23]: Thank you! @iskerrett @knolleary

**Pull request merged**